web/pdf_viewer.css

```
@@ -30,7 +31,8 @@
   :root {
     --pdfViewer-padding-bottom: 9px;
     --page-margin: 9px auto 0;
```

*I've not tested, but simply reading the surrounding the code:*
Don't you need to change these values, to account for the fact that there's now a border added just below?
Comparing with the "normal" rules, should this be updated to e.g. the following now?
```suggestion
    --page-margin: 8px auto -1px;
```